src/patternfly/components/Table/table.scss

```diff
@@ -303,14 +303,14 @@
 
 // Check table cell
 // ==================================================================
-.pf-c-table__check {
-  vertical-align: top;
-
-  > * {
-    margin-top: var(--pf-c-table__check--input--MarginTop);
-    vertical-align: top;
-  }
-}
+// FIX #1577: Remove this to fix the table checkbox alignment
```

Comment:
We don't comment out blocks (if it's not needed, you can just remove it), and ideally we would be able to test the change in the PR build preview that is posted as a comment to this PR (https://deploy-preview-2143--pf-next.netlify.com/), and not have to build it locally and uncomment the code. Can you go ahead and uncomment it?

Reply:
Thanks for the review. I've removed the commented code out.